subgraphs/_reference_/src/prices/common/constants.ts

```diff
@@ -6,12 +6,29 @@ import { SushiSwapPair__getReservesResult } from "../../../generated/UniswapV2Fa
 /////////////////////////////////// COMMON ////////////////////////////////
 ///////////////////////////////////////////////////////////////////////////
 
+export namespace Network {
+  export const ARBITRUM_ONE = "arbitrum-one";

```

Comment:
these gotta be upper case for enums otherwise graph-node will complain. You can convert to lower when using them